src/cmd/sign.js

```diff
@@ -13,12 +13,46 @@ import {createLogger} from '../util/logger';
 const log = createLogger(__filename);
 export const extensionIdFile = '.web-extension-id';
 
-export default function sign(
-    {verbose, sourceDir, artifactsDir, apiKey, apiSecret,
-     apiUrlPrefix, id, timeout}: Object,
-    {build=defaultBuilder, signAddon=defaultAddonSigner,
-     preValidatedManifest=null}: Object = {}): Promise<Object> {
+// Flow Types
+
+import type {ExtensionManifest} from  '../util/manifest';
+
+export type SignCmdParams = {
```

Comment:
nit: could we call these `SignParams` and `SignOptions`? The `sign()` function could be used by a library -- i.e. not as a command.